db/seeds.rb

```diff
@@ -28,6 +29,9 @@
 pull_toy = dog_shop.items.create(name: "Pull Toy", description: "Great pull toy!", price: 10, image: "http://lovencaretoys.com/image/cache/dog/tug-toy-dog-pull-9010_2-800x800.jpg", inventory: 32)
 dog_bone = dog_shop.items.create(name: "Dog Bone", description: "They'll love it!", price: 21, image: "https://img.chewy.com/is/image/catalog/54226_MAIN._AC_SL1500_V1534449573_.jpg", active?:false, inventory: 21)
 
+# delete item
+delete_item = bike_shop.items.create(name: "Mike Dao #1", description: "What an A plus project!", price: 10, image: "https://pbs.twimg.com/profile_images/1219492204193779712/-W-CA2jc_400x400.jpg", inventory: 2)
```

Comment:
Mike Dao item to show that an item can be deleted

Reply:
![](https://media2.giphy.com/media/3oEjHI8WJv4x6UPDB6/giphy.gif?cid=5a38a5a27a622fad7ac8debf43e03e11dcaff4551c3a2928&rid=giphy.gif)